contracts/VToken.sol

```diff
@@ -818,6 +1093,82 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
         return actualRepayAmount;
     }
 
+    /**
+     * @dev Allows a borrower to swap his debt between stable and variable mode, or vice versa
+     * @param rateMode The rate mode that the user wants to swap to
+     **/
+    function swapBorrowRateMode(uint256 rateMode) external {
+        accrueInterest();
```

Comment:
Would it make sense to invoke a pre-hook in the comptroller, checking for example if the swap is paused? Just I was wondering. Of course, no blocker for the PR, and probably it's more a Product decision

Reply:
Yes, I was also thinking the same while implementing the functionality. I have included preHook, paused check for swap.
Can verify the changes at [commit](https://github.com/VenusProtocol/isolated-pools/pull/132/commits/0f14951544955613711dd9321a85af4f96073049).